server/src/index.js

```diff
@@ -52,10 +53,15 @@ const server = new ApolloServer({
 const app = express();
 const cors = require("cors");
 app.use(cors());
+app.get(/^((?!graphql).)*$/,
```

Comment:
Can you add a comment to explain this a bit?

Reply:
Yes, sure, did